src/components/newMeasure/CheckBox.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Checkbox as MUICheckBox } from "@mui/material";
+import { styled } from "@mui/material/styles";
+
+const CheckBox = styled(MUICheckBox)(() => ({
+  padding: "1px 4px",
+}));
+
+export default function StyledCustomization(props) {
+  return <CheckBox {...props} disableRipple />;
+}
```

Comment:
Can we move this to madie design system ? so that we can reuse it.

Reply:
We could but the changes are so minor to the point it's just minimal spacing, and I'm unsure we will have this small double checkbox format anywhere else in any of the applications.

Reply:
ok Make sense !!